apps/homepage/__tests__/e2e/test.cy.tsx

```diff
@@ -1,16 +1,64 @@
-describe('First test', () => {
-  it('Should check Header see through out pages', () => {
-    cy.visit('/');
+describe('Localization Route Tests', () => {
+  const locales = ['en', 'es'];
+  const routes = ['/', '/projects', '/contact'];
 
-    cy.findByText("Header").should('be.visible');
+  locales.forEach((locale) => {
+    routes.forEach((route) => {
+      it(`should display the ${route} page correctly in ${locale}`, () => {
+        cy.visit(`/${locale}${route}`);
```

Comment:
We should test that it goes automatically to this route, rather than hardcoding the `{route}`

Reply:
This is not resolved yet.

Reply:
Look for `headers`: https://docs.cypress.io/api/commands/visit#Arguments